src/main/java/com/gamegoo/gamegoo_v2/email/service/EmailService.java

```diff
@@ -0,0 +1,112 @@
+package com.gamegoo.gamegoo_v2.email.service;
+
+import com.gamegoo.gamegoo_v2.email.domain.EmailVerifyRecord;
+import com.gamegoo.gamegoo_v2.email.repository.EmailVerifyRecordRepository;
+import com.gamegoo.gamegoo_v2.exception.EmailException;
+import com.gamegoo.gamegoo_v2.exception.common.ErrorCode;
+import com.gamegoo.gamegoo_v2.utils.EmailTemplateProcessor;
+import com.gamegoo.gamegoo_v2.utils.RandomCodeGeneratorUtil;
+import jakarta.mail.internet.MimeMessage;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Service
+@RequiredArgsConstructor
+@Transactional
+@Slf4j
+public class EmailService {
+
+    private final EmailVerifyRecordRepository emailVerifyRecordRepository;
+    private final JavaMailSender javaMailSender;
+
+    /**
+     * 랜덤 코드 이메일 전송
+     *
+     * @param email 이메일 주소
+     */
+    public void sendEmailVerificationCode(String email) {
+        // 해당 이메일로 3분 이내에 3번 이상 요청을 보냈을 경우 제한
+        checkEmailSendRequestLimit(email);
+
+        // 랜덤 코드 생성하기
+        String certificationNumber = RandomCodeGeneratorUtil.generateEmailRandomCode();
+
+        // HTML 템플릿 경로
+        String templatePath = "templates/email-template.html";
+
+        // Placeholder 값 정의
+        Map<String, String> placeholders = new HashMap<>();
+        placeholders.put("CERTIFICATION_NUMBER", certificationNumber);
+
+        // 이메일 전송
+        sendEmail(email, "GameGoo 이메일 인증 코드", templatePath, placeholders);
+
+        // 이메일 전송 기록 DB 저장
+        EmailVerifyRecord emailVerifyRecord = EmailVerifyRecord.create(email, certificationNumber);
+        emailVerifyRecordRepository.save(emailVerifyRecord);
```

Comment:
```suggestion
        emailVerifyRecordRepository.save(EmailVerifyRecord.create(email, certificationNumber));
```
이렇게 한줄로 작성할 수 있을 것 같아요